cleanlab/models/keras.py

```diff
@@ -75,10 +75,12 @@ def __init__(
         compile_kwargs: dict = {
             "loss": tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
         },
+        params: dict = {},
     ):
         self.model = model
         self.model_kwargs = model_kwargs
         self.compile_kwargs = compile_kwargs
+        self.params = params
```

Comment:
```suggestion
        self.params: dict = params or {}
```

You can ignore the type-hint if you want.